Libraries/LibWeb/Editing/Commands.cpp

```diff
@@ -509,37 +509,37 @@ enum class FontSizeMode : u8 {
 };
 
 // https://w3c.github.io/editing/docs/execCommand/#the-fontsize-command
-bool command_font_size_action(DOM::Document& document, String const& value)
+bool command_font_size_action(DOM::Document& document, Utf16View value)
 {
     // 1. Strip leading and trailing whitespace from value.
-    auto resulting_value = MUST(value.trim_ascii_whitespace());
+    auto resulting_value = value.trim_whitespace();
```

Comment:
oh maybe we should rename `Utf16View::trim_whitespace` to `Utf16View::trim_ascii_whitespace` to make it clear it's ASCII since, like `String`, it's supposed to be Unicode-aware unless otherwise mentioned.